quiet_logistics_endpoint.rb

```diff
@@ -89,6 +89,19 @@ class QuietLogisticsEndpoint < EndpointBase::Sinatra::Base
     result code, message
   end
 
+  post '/cancel_shipment_order' do
+    begin
+      order   = @payload['order']
+      message = Api.send_document('ShipmentOrderCancel', order, outgoing_bucket, outgoing_queue, @config)
```

Comment:
'ShipmentOrderCancel' isn't on the case statement in Api; apparently it should be 'CancelOrder':
```
when 'CancelOrder'
        Documents::ShipmentOrderCancel.new(content, config)
```

Reply:
Awesome catch, I will stick to ShipmentOrderCancel that is the one QL documentation uses. Tks!

Reply:
Updated on the api file.